### ISSUE TITLE
EXCUSE ME?

### ISSUE THREAD

**Issue by LeOofDesignsWasTaken** [Honguito98/batch-game-batchcraft#2]
this clearly deletes a registry key! i looked in the program code and look what i found! 
'set "cmd.con=HKCU\Console\%%SystemRoot%%_system32_cmd.exe /v"
Reg export HKCU\Console Backup.reg>nul
Reg delete HKCU\Console\%%SystemRoot%%_system32_cmd.exe /f>nul`
the "Reg delete HKCU\Console\%%SystemRoot%%_system32_cmd.exe /f>nul" part is the registry key deletion part

**Comment by anic17** [comment]
@LeOofDesignsWasTaken This is a user registry relating to console settings. It is deleting a registry key, but `HKCU\Console` is probably the safest key to modify as it only affects console.
And, as you can also see, it's making a backup of your registry in case that something goes wrong.

Finally, I just want to let you know that today all programs modify registry, and they don't make your computer crash. Please get informed before starting an issue.

**Comment by HHonzik** [comment]
> this clearly deletes a registry key! i looked in the program code and look what i found! 'set "cmd.con=HKCU\Console%%SystemRoot%%_system32_cmd.exe /v" Reg export HKCU\Console Backup.reg>nul Reg delete HKCU\Console%%SystemRoot%%_system32_cmd.exe /f>nul` the "Reg delete HKCU\Console%%SystemRoot%%_system32_cmd.exe /f>nul" part is the registry key deletion part

That can't damage windows or your user files, it only just deletes a registry key and after more line of code it puts back if you want to see the code then here:
Reg import Backup.reg>nul
that puts back the deleted registry key

**Comment by ctrl-alt-dale** [comment]
its for modifying Command Prompt aesthetics

**Comment by Honguito98** [comment]
Hi.
Sorry for the delayed response.

As you could read, it effectively does create a small backup of that registry key before making any changes.
It is done to modify the console font size.

This in practice **cannot** damage your registry, but I agree that there should be a better aproach to make it more stable, specially, in theorical, very edge case where, lets say, you got a power outrage at the very same time you run the file, in that case you could run the 'Console Backup.reg' file located in the same dir as the BAT file (or at C:\WINDOWS\system32 if run with Admin rights)

I will close this issue since this does not causes any problem right now,
but I will open a new issue regarding this future improvement.
